Cargo.toml

```diff
@@ -173,7 +173,7 @@ rpath = false
 # For example:
 # arrow-format = { git = "https://github.com/datafuse-extras/arrow-format", rev = "78dacc1" }
 
-arrow2 = { git = "https://github.com/jorgecarleitao/arrow2", rev = "615300b" }
+arrow2 = { git = "https://github.com/TCeason/arrow2", branch = "ISSUE-1411" }
```

Comment:
Please merge this change into datafuse-extras/arrow2 instead.

Reply:
Yes, this is a test for https://github.com/jorgecarleitao/arrow2/pull/1412, when this pr merged, I will request review again.